gulp-tasks/build-packages.js

```diff
@@ -142,22 +143,59 @@ const buildPackage = (packagePath, buildType) => {
   .pipe(gulp.dest(outputDirectory));
 };
 
+const buildNodePackage = (packagePath) => {
+  const packageName = pkgPathToName(packagePath);
+  const outputDirectory = path.join(packagePath,
+    constants.PACKAGE_BUILD_DIRNAME);
+
+  logHelper.log(oneLine`
+    Building Node Package for
+    ${logHelper.highlight(packageName)}.
+  `);
+
+  return gulp.src(`${packagePath}/src/**`).pipe(babel({
+    only: /\.js$/,
+    presets: [
+      ['env', {
+        targets: {
+          // Change this when our minimum required node version changes.
+          node: '4.0',
+        },
+      }],
+    ],
+    plugins: [
+      'transform-runtime',
+    ],
+  })).pipe(gulp.dest(outputDirectory));
+};
+
 const cleanPackages = (packagePath) => {
   const outputDirectory = path.join(packagePath,
-    constants.PACKAGE_BUILD_DIRNAME, constants.BROWSER_BUILD_DIRNAME);
+    constants.PACKAGE_BUILD_DIRNAME);
   return fs.remove(outputDirectory);
 };
 
 gulp.task('build-packages:clean', gulp.series(
-  packageRunnner('build-packages:clean', cleanPackages)
+  packageRunnner('build-packages:clean', {
```

Comment:
I'm struggling to understand what this is going with the NODE and BROWSER variants (Splitting up relevant stuff into two seperate tasks would help).